test/e2e/gardener/shoot/shoot.go

```diff
@@ -265,3 +272,156 @@ func ItShouldAnnotateShoot(s *ShootContext, annotations map[string]string) {
 		}).Should(Succeed())
 	}, SpecTimeout(time.Minute))
 }
+
+// ItShouldLabelManualInPlaceNodesWithSelectedForUpdate labels all manual in-place nodes with the selected-for-update label.
+// In the actual scenario, this should be done by the user, but for testing purposes, we do it here.
+func ItShouldLabelManualInPlaceNodesWithSelectedForUpdate(s *ShootContext) {
+	GinkgoHelper()
+
+	It("should label all the manual in-place nodes with selected-for-update", func(ctx SpecContext) {
+		for _, pool := range s.Shoot.Spec.Provider.Workers {
+			if !v1beta1helper.IsUpdateStrategyManualInPlace(pool.UpdateStrategy) {
+				continue
+			}
+
+			nodeList := &corev1.NodeList{}
+			Eventually(ctx, s.ShootKomega.List(nodeList, client.MatchingLabels{v1beta1constants.LabelWorkerPool: pool.Name})).Should(Succeed())
+
+			for _, node := range nodeList.Items {
+				if metav1.HasLabel(node.ObjectMeta, machinev1alpha1.LabelKeyNodeSelectedForUpdate) {
+					continue
+				}
+
+				Eventually(ctx, s.ShootKomega.Update(&node, func() {
+					metav1.SetMetaDataLabel(&node.ObjectMeta, machinev1alpha1.LabelKeyNodeSelectedForUpdate, "true")
+				})).Should(Succeed())
+			}
+		}
+	}, SpecTimeout(2*time.Minute))
+}
+
+// ItShouldFindAllMachinePodsBefore finds all machine pods before running the required tests and returns their names.
+func ItShouldFindAllMachinePodsBefore(s *ShootContext) sets.Set[string] {
+	GinkgoHelper()
+
+	beforeStartMachinePodNames := sets.New[string]()
+
+	It("Find all machine pods to ensure later that they weren't rolled out", func(ctx SpecContext) {
```

Comment:
same for all its

```suggestion
	It("find all machine pods to ensure later that they weren't rolled out", func(ctx SpecContext) {
```

Reply:
This is inconsistent, then, why some of them have capital letter and some don't.